packages/driver/cypress/e2e/dom/visibility.cy.ts

```diff
@@ -819,10 +819,9 @@ describe('src/cypress/dom/visibility', () => {
     })
 
     describe('css overflow', () => {
-      it('is visible when parent doesnt have overflow hidden', function () {
-        expect(this.$parentNoWidthHeightOverflowAuto.find('span')).to.be.visible
-
-        expect(this.$parentNoWidthHeightOverflowAuto.find('span')).to.not.be.hidden
+      it('is hidden when parent overflow auto and no width/height', function () {
```

Comment:
Is this removed test or some form of it no longer valid?

Reply:
The name & test previously written was incorrect. The name update reflect other naming in this file,

The element being tested is:

```js
      this.$parentNoWidthHeightOverflowAuto = add(`\
<div style='width: 0; height: 0px; overflow: auto;'>
  <span>parent no size, overflow: auto</span>
</div>`)
```
is not actually visible. See screenshot:

<img width="1412" alt="Screen Shot 2023-10-26 at 1 15 42 PM" src="https://github.com/cypress-io/cypress/assets/14099737/705245fb-e304-42db-87cc-defb4a18083a">